lib/services/oauth/oauth_server.go

```diff
@@ -64,6 +68,22 @@ func GetClientStore() *pg.ClientStore {
 }
 
 func passwordAuthorizationHandler(ctx context.Context, clientID, email string, password string) (string, error) {
-	// TODO: Implement the logic in Sign In task (#26)
-	return "1", nil
+	db := database.GetDB()
+
+	var user models2.User
+	db.Where("email = ?", email).First(&user)
+	if user.Base.ID.ID() == 0 {
```

Comment:
```suggestion
	result := db.Where("email = ?", email).First(&user)
	if result.Error != nil {
```

Go loves errors :D There should be a NotFound error in the retuned value :)
Besides, that avoid the strange `ID.ID() == 0` (ID is a `UUID` so we should compare with a nil uuid).

In case you need later a nil UUID comparision, the documentation offers some insights:

```
if user.Base.ID == uuid.Nil {
```
That would work too (but error check is preferred)